### PR TITLE
Export number of frost and heat events

### LINKED ISSUE #9230: Exports frost and heat event number in the damage function
### Describe the new feature

In the damage functions for wheat and canola, export new variables for number of frost and heat events.

* `FrostEventNo` for number of frost events during sensitive period
* `HeatEventNo` for number of heat events during sensitive period

### PR DESCRIPTION
**Pull request by byzheng**
Resolves #9230

### CODE REVIEW

**Comment by byzheng** [comment]
@PengchengHU Please review my changes here.

**Comment by BrianCollinss** [comment]
@byzheng does this algorithm not affect the internal processes, or does it just calculate the stressed yield as a standalone indicator without affecting biomass partitioning, etc?

**Comment by PengchengHU** [comment]
Hi Brian,

It is a post-process on the APSIM-simulated yield and has nothing to do
with the internal processes.

Best Regards,
Pengcheng HU

File: Models/Functions/FrostHeatDamageFunctions.cs [hunk 1]
```diff
@@ -193,6 +195,12 @@ public CropTypes CropType
         /// <summary>Cumulative actual yield reduction ratio induced by heat stress.</summary>
         public double CumulativeHeatReductionRatio { get; set; }
 
+        /// <summary>Number of frost events during sensitive period.</summary>
+        public double FrostEventNO { get; set; }
+
+        /// <summary>Number of heat events during sensitive period.</summary>
+        public double HeatEventNo { get; set; }
```
Comment (hol353) [inline Models/Functions/FrostHeatDamageFunctions.cs:202]: Can you rename these two properties to ```FrostEventNumber``` and ```HeatEventNumber``` ?

**Comment by byzheng** [comment]
Variable renamed

**Pull request merged**